src/sdk/model/json/perf-samples.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * API PerfSampleSerie JSON Representation.
+ */
+export interface PerfSampleSerie {
```

Comment:
all of our current JSON interface representations have names that end in `Json`, lets not deviate from that naming precedent.